lib/index.d.ts

```diff
@@ -0,0 +1,71 @@
+// TODO: Add missing type definitions for all import/export
```

Comment:
Most of the exports can be exported directly from victory module instead of the `./component` version because the victory component doesn't change its props signature.  In the few cases where it does, I've created individual declaration files for each.